### ISSUE TITLE
Missing artifact icons for some artifacts

### ISSUE THREAD

**Issue by krs0** [vcmi/vcmi#1798]
**Describe the bug**
Some artifacts have only placeholder icons

**To Reproduce**
1. use cheat vcmiforgeofnoldorking
2. browse artifact to the very end
Or transfer between 2 heroes to have them rearranged faster.

**Screenshots**
![image](https://user-images.githubusercontent.com/15194321/228598426-671d4da9-f8da-480c-b7eb-396ef1e5f8a8.png)

**Version**
 - OS: Windows
 - Version: beta

**Comment by rilian-la-te** [comment]
@krs0 this artifacts is implemented only in hota. In OH3 there artifacts was always banned due to lack of implementation.

**Comment by krs0** [comment]
But I do not have HotA mod :)

**Comment by misiokles** [comment]
I suspect krs0 plays some SoD maps in which such placeholder artifacts were unlocked with some tricks.

**Comment by rilian-la-te** [comment]
@misiokles no, he is just use forgeofthenoldorking. I noticed this behaviour when used same cheat myself to test archery artifacts.

**Comment by GeorgeK1ng** [comment]
This issue is still there when using **vcmiartifacts**, maybe we need to directly ban these artifact with special flag?

**Comment by IvanSavenko** [comment]
Those artifacts already have special class. Probably cheat should exclude such artifacts.
